ecocode-rules-specifications/pom.xml

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>io.ecocode</groupId>
+		<artifactId>ecocode-parent</artifactId>
+		<version>1.2.2-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>ecocode-rules-specifications</artifactId>
+
+	<name>ecoCode Rules Specifications repository</name>
+	<description>Repository that contains the specifications of every static-analysis rules available in ecoCode plugins.</description>
+
+	<properties>
```

Comment:
delete this `properties` block because sourceEncoding is already present on parent pom